src/models/applicationLine.model.js

```diff
@@ -34,6 +35,22 @@ module.exports = class ApplicationLine extends BaseModel {
     }
   }
 
+  static async getValidRulesetIds (authToken, applicationLineId) {
+    const dynamicsDal = new DynamicsDalService(authToken)
+    const rulesetIds = Object.keys(RulesetIds).map((prop) => RulesetIds[prop])
+    const query = encodeURI(`defra_applicationlines(${applicationLineId})?$expand=defra_parametersId($select=${rulesetIds.join()})`)
+    try {
+      const result = await dynamicsDal.search(query)
+      if (result && result.defra_parametersId) {
+        // return only those rulesetIds with a value of true
+        return rulesetIds.filter((rulesetId) => result.defra_parametersId[rulesetId])
```

Comment:
I would prefer the return value variable to be declared outside the if statement and then always returned

Reply:
Changed as requested